app/scripts/views/settings/settings-file-view.js

```diff
@@ -160,7 +160,16 @@ const SettingsFileView = Backbone.View.extend({
                 return;
             }
         }
-        this.appModel.syncFile(this.model, arg);
+
+        const password = this.$el.find('#settings__file-master-pass').val();
```

Comment:
This should be somehow properly refactored≥

Reply:
I just wanted to merge this for the next release which should be coming soon, but noticed this. It's something that's better not to have in master.

Reply:
I removed fingerprint update call since I didn't find another suitable workaround.
User will be asked to protect password with fingerprint on next login after password change.